.github/workflows/test.yml

```
@@ -63,7 +63,7 @@ jobs:
         run: |
           go mod verify
           go mod download
-          go test -v ./...
+          go test -race -coverprofile=coverage.txt -covermode=atomic ./...
```

```suggestion
          go test -v -race ./...
```
Windows does not seem to like filenames without quotes `"`. I think we don't need to compute the code coverage numbers on multiple platforms; the coverage should be the same. But adding `-race` flag is probably a good change here as the behavior of the go routines might differ between platforms..